pypxe-server.py

```diff
@@ -137,7 +153,8 @@
                     useipxe = args.USE_IPXE,
                     usehttp = args.USE_HTTP,
                     mode_proxy = args.DHCP_MODE_PROXY,
-                    mode_debug = ("dhcp" in args.MODE_DEBUG.lower() or "all" in args.MODE_DEBUG.lower()))
+                    mode_debug = ("dhcp" in args.MODE_DEBUG.lower() or "all" in args.MODE_DEBUG.lower()),
```

Comment:
Unless I am completely mistaken as to what rebase does, the reason this line is highlighted is becuase I'm adding an argument after it, changing the last parenthesis to a comma.